airflow/models.py

```diff
@@ -1383,14 +1383,41 @@ def run(
         settings.engine.dispose()
         if verbose:
             if mark_success:
-                msg = "Marking success for "
```

Comment:
I don't think abstracting everything into a variable is cleaner at all. Think about fizzbuzz where you print fizz and buzz in the i%3 and i%5 case, opposed to the printing fizz in i%3, buzz in i%5, and fizzbuzz in i%15. The second way is way cleaner.